packages/common/src/discussions/utils.ts

```diff
@@ -27,11 +27,11 @@ export function setDiscussableKeyword(
   typeKeywords: string[],
   discussable: boolean
 ): string[] {
-  const updatedTypeKeywords = typeKeywords.filter(
+  const updatedTypeKeywords = typeKeywords?.filter(
     (typeKeyword: string) => typeKeyword !== CANNOT_DISCUSS
   );
   if (!discussable) {
-    updatedTypeKeywords.push(CANNOT_DISCUSS);
+    updatedTypeKeywords?.push(CANNOT_DISCUSS);
   }
-  return updatedTypeKeywords;
+  return updatedTypeKeywords || [];
```

Comment:
Minor, we could eliminate the optional chaining operators and the `|| []` in the return expression by refactoring slightly to:

```ts
export function setDiscussableKeyword(
  typeKeywords: string[],
  discussable: boolean
): string[] {
  const updatedTypeKeywords = (typeKeywords || []).filter(
    (typeKeyword: string) => typeKeyword !== CANNOT_DISCUSS
  );
  if (!discussable) {
    updatedTypeKeywords.push(CANNOT_DISCUSS);
  }
  return updatedTypeKeywords;
}
```